controls/radmulticolumncombobox/gridview-itemssourceprovider.md

```diff
@@ -59,6 +59,16 @@ The visibility of the column headers and footers can be controlled through the f
 
 Since __R1 2019 SP1__, the GridViewItemsSourceProvider exposes the __IsReadOnly__ property through which you can control the __IsReadOnly__ property of the RadGridView control in the dropdown.
 
+## RowIndicatorVisibility
+
+> More information can be found in the [Hiding the Row Indicator]({%slug gridview-customizing-rows%}) topic.
+
+By default the first cell of a row represents row indicator, which appears when the row is set as current. Since __R2 2019__, the __GridViewItemsSourceProvider__ exposes the __RowIndicatorVisibility__ property which controls the visibility of the row indicator.
+
+## ContextMenu
+
+In __R2 2019__, we introduced the __ContextMenu__ property of the __GridViewItemsSourceProvider__ which allows you to set a custom ContextMenu.

```

Comment:
Usually, we avoid such language in the docs :D. I mean statements from personal point of view, like: "we introduced this", "we did that", "now let's define this class". 

This is an instructional documentation which means that we can use (and in my opinion it is recommended to do so) direct instructions like: "to do this, set that property" or "step 2: define the MainViewModel class", or "In R2 2019 the ContextMenu property of GridViewItemsSourceProvider was introduced".

For this concrete case, I would describe the property and note that it was released in R2 2019 separately. For example: 
______
Note: (with the > bracket) This feature was introduced with R2 2019.

To set a custom context menu, set the ContextMenu property of GridVIewItemsSourceProvider
[code snippet here]
[picture here]
____